src/v2/spec.yaml

```diff
@@ -2102,6 +2144,51 @@ paths:
           $ref: "#/components/responses/400Response"
         "500":
           $ref: "#/components/responses/500Response"
+  /farcaster/conversation:
+    get:
+      tags:
+        - Cast
+      summary: Retrieve conversations above and below a given hash or Warpcast URL
+      description: Gets information about an conversations surrounding a cast by passing in a Warpcast web URL or cast hash
+      externalDocs:
+        url: https://docs.neynar.com/reference/cast
+      operationId: conversation
+      parameters:
+        - $ref: "#/components/parameters/ApiKey"
+        - name: identifier
+          required: true
+          in: query
+          example: "https://warpcast.com/rish/0x9288c1"
+          description: Cast identifier (Its either a url or a hash)
+          schema:
+            type: string
+        - name: type
+          required: true
+          in: query
+          example: url
+          schema:
+            $ref: "#/components/schemas/CastParamType"
+        - name: levels
+          in: query
+          required: false
+          description: "levels up and down to fetch conversations"
```

Comment:
we should add the min and max in the description so looking at it, developers know what the range is

Reply:
this should be in the Levels object @rishavmukherji